.cookiecutter/includes/gitignore

```diff
@@ -1,2 +1,12 @@
 via/static/**/*.gz
 build.tar
+
+# See https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored.
+# This is the "If you're not using Zero-Installs" list.
+.pnp.*
+.yarn/*
+!.yarn/patches
+!.yarn/plugins
+!.yarn/releases
+!.yarn/sdks
+!.yarn/versions
```

Comment:
These lines had been added to `.gitignore` directly, moved them into this include so that `make template` doesn't delete them